test_suite/conftest.py

```diff
@@ -33,6 +35,20 @@ def check_markers(host, request):
     host_version = host.system_info.release
     distro_version = f'{host_distro}{host_version}'
 
+    # Skip the test if the distro is explicitely excluded
+    if exclude_on_marker:
+        exclude_on_marker_list = exclude_on_marker.args[0]
+        if host_distro in exclude_on_marker_list \
+                or distro_version in exclude_on_marker_list:
+            pytest.skip(f"This test doesn't apply to {distro_version}")
```

Comment:
```suggestion
            pytest.skip(f'This test doesn't apply to {distro_version}')
```
We adopted the convention of using single quotes whenever possible (although linters do not care about it).

Reply:
Oh I didn't realize that as this part of the code contains the double quotes  on multiple occasions, will keep that in mind!

Reply:
Oh, it's true that in this case the double quotes do make sense! Since the "doesn't" apostrophe is there.